docs/ssh/main.md

```diff
@@ -82,11 +82,14 @@ x
 !!! note
 	Login nodes have a small storage space for users to store a very small volume of data required for back-end processes such as authentication and other system-related processes upon logging in. Login nodes are not a storage space to save and install our packages. 
 	
-!!! SSH key-based authentication
-    In compliance with the University of Chicago security guidelines, 2FA is required with limited exceptions. If you believe you have a justifiable need for SSH key pairs (only PIs), please [contact our helpdesk](https://rcc.uchicago.edu/support-and-services/consulting-and-technical-support){:target='_blank'} and describe your situation. Once your request is received, the RCC security team will review it, and we will follow up with you as soon as possible. 
+!!! note
+    In compliance with the University of Chicago security guidelines, 2FA is required with limited exceptions. If you believe you have a justifiable need for SSH key-based authentication (only PIs), please [contact our helpdesk](https://rcc.uchicago.edu/support-and-services/consulting-and-technical-support){:target='_blank'} and describe your situation. Once your request is received, the RCC security team will review it, and we will follow up with you as soon as possible. 
+
+!!! note
+    There are 2 or more login nodes on each cluster to share the workload from all the login users. You can land on one of the login nodes depending on their workload condition at the time. You may choose to log in a specific login node, e.g. `midway2-login2.rcc.uchicago.edu`, but it is typically not necessary.
```

Comment:
- "workload condition" => "workload conditions"?
- "log in a" => "log into a"?
- "but it is typically not necessary." => "but it is typically discouraged."

Reply:
fixed